Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -2361,17 +2361,24 @@ currentLocationView push state =
 
 nearByPickUpPointsView :: forall w . HomeScreenState -> (Action -> Effect Unit) -> PrestoDOM (Effect Unit) w
 nearByPickUpPointsView state push =
-  linearLayout
-    [ height WRAP_CONTENT
+  scrollView
+    [ height (V 130)
```

Comment:
don't hard code the height try to use `getLayoutBounds` to update layout bounds dynamically.

Reply:
done